project/app/src/main/java/org/owntracks/android/services/ServiceMessageHttp.java

```diff
@@ -14,7 +14,7 @@
 import com.google.android.gms.gcm.OneoffTask;
 import com.google.android.gms.gcm.Task;
 
-import org.antlr.v4.runtime.misc.NotNull;
```

Comment:
This package was flagged as non-existing in my build, even with just the baseline upstream/master repo without my changes.